_data/banking.yml

```
@@ -606,6 +752,8 @@ websites:
       - phone
       - email
     doc: https://www.firstrepublic.com/resource/banking-online-faqs#su10
```

```suggestion
    doc: https://www.firstrepublic.com/resource/entering-a-6-digit-code-at-every-sign-in
```
Doc link gives 404. Replacement is the only 2FA-related link I could find on the site.